### ISSUE TITLE
Project Halted?

### ISSUE THREAD

**Issue by Homerade** [richeyb/greenfy-example#1]
Not planning on building this out??

I just stumbled on this today, loved the way you explain things and following along with the timber post. 

Any plans on picking it back up? Looking for more 👀

**Comment by lonniev** [comment]
Agreed, it would be good to see Part Two where Create, Update, and Delete topics get addressed if not yet implemented.
